src/RefractAST.cc

```diff
@@ -315,36 +327,82 @@ namespace drafter
                 }
             }
 
-            if (IElement* e = SimplifyRefractContainer(samples)) {
-                element->attributes["sample"] = e;
-            }
+        }
 
-            if (IElement* e = SimplifyRefractContainer(defaults)) {
-                element->attributes["default"] = e;
-            }
+        if (IElement* e = SimplifyRefractContainer(samples)) {
+            element->attributes["sample"] = e;
```

Comment:
see my comment above on using constants – this should be really defined just once.